### DISCUSSION TITLE
Ollama as a separate container

### DISCUSSION THREAD

**Discussion by mrepetto94** [imartinez/privateGPT#1761] [Q&A]
Has anyone tried to run Ollama in a separate container with docker-compose?

Changing the `api_base` to `http://ollama:11434` does not return anything aside from this warning:
```
[WARNING ] llama_index.core.chat_engine.types - Encountered exception writing response to history: [Errno 99] Cannot assign requested address
```

**Comment by alptekin** [comment]
hello could you find any solution. I have the same problem.

**Comment by mrepetto94** [comment]
Look at pull request #1812
________________________________
Da: Alptekin Topal ***@***.***>
Inviato: Monday, April 1, 2024 10:23:58 AM
A: zylon-ai/private-gpt ***@***.***>
Cc: Marco Repetto ***@***.***>; Author ***@***.***>
Oggetto: Re: [zylon-ai/private-gpt] Ollama as a separate container (Discussion #1761)


hello could you find any solution. I have the same problem.
